docs/rules/jsx-first-prop-new-line.md

```diff
@@ -2,13 +2,21 @@
 
 Ensure correct position of the first property.
 
+**Fixable:** This rule is automatically fixable using the `--fix` flag on the command line.
+
 ## Rule Details
 
 This rule checks whether the first property of all JSX elements is correctly placed. There are three possible configurations:
 * `always`: The first property should always be placed on a new line.
 * `never` : The first property should never be placed on a new line, e.g. should always be on the same line as the Component opening tag.
 * `multiline`: The first property should always be placed on a new line when the properties are spread over multiple lines.
 
+In order to utilise autofix, please specify the indentation style as an additional argument in your configuration:
```

Comment:
If we are to rely on `indent` autofixer wouldnt we require 2 runs of lint?
Anyway the current implementation does default to a specific style (2 spaces). Most of the indent fixing code is actually referenced and stripped down from `jsx-indent`.

To make it clearer, I could word it such that autofix has a optional arg for indents *but* does work without it.

PS: I think I should change the default back to 4 spaces so to be inline with `indent` defaults